lib/miq/lwimiq.rb

```diff
@@ -98,5 +103,13 @@ def uri_for(repo)
         ].join("&")
       )
     end
+
+    def file_rename
+      @subtitle == '<subtitle>' ? 'CHANGE-ME' : cleanup
```

Comment:
Maybe we could set these defaults at construction time?

Reply:
Do you think we should handle all the logic at construction time?  I remember going back and forth - but decided on this direction to keep the logic away from the options.